libvast/src/system/partition.cpp

```diff
@@ -65,19 +65,39 @@ using namespace caf;
 
 namespace vast::system {
 
-namespace {
-
-// The functions in this namespace take PartitionState as template argument
-// because the impelementation is the same for passive and active partitions.
+caf::actor active_partition_state::indexer_at(size_t position) const {
+  VAST_ASSERT(position < indexers.size());
+  return as_vector(indexers)[position].second;
+}
 
 /// Gets the INDEXER at a certain position in the `indexers` stable map.
```

Comment:
I think the comment no longer holds, right?

Reply:
Right, updated.